lib/ex_unit/lib/ex_unit/callbacks.ex

```diff
@@ -12,10 +13,10 @@ defmodule ExUnit.Callbacks do
   structured value (see below).
 
   The `setup_all` callbacks are invoked only once per module, before any
-  test runs. All `setup` callbacks are run before each test. No callback
-  runs if the test case has no tests or all tests have been filtered out.
+  test is run. All `setup` callbacks are run before each test. No callback
+  is run if the test case has no tests or all tests have been filtered out.
 
-  Both `setup` and `setup_all` can be defined by a block, by passing
+  `setup/1` and `setup_all` callbacks can be defined by a block, by passing
```

Comment:
When do you chose to add the arity or not? `setup` has an arity but `setup_all` doesnt.

Reply:
It's a mistake. It should have no arity. I will check it once im with my computer

Reply:
done